relayer/src/foreign_client.rs

```diff
@@ -977,7 +977,7 @@ impl<DstChain: ChainHandle, SrcChain: ChainHandle> ForeignClient<DstChain, SrcCh
 
         // Wait for the source network to produce block(s) & reach `target_height`.
         while src_application_latest_height()? < target_height {
-            thread::sleep(Duration::from_millis(100))
+            thread::sleep(Duration::from_millis(100));
```

Comment:
Same here, but since this is on the relay path it makes sense to have a smaller interval than in the CLI, perhaps 500ms?

Reply:
I suspect the caching layer will take the brunt of the call to `src_application_latest_height` (cache expires every 200ms or so). So from a relay path perspective, not sure if we should make this value 500ms, we can probably leave it as is. I recall having this discussion with Anca: whether we shouldn't increase the value here, and the argument was that the `latest_height` query is the cheapest and we can afford to retry often. Before the caching layer we were retrying every 100ms; after we introduced the caching layer that means we're retrying every ~250ms or so (backoff 100ms from the sleep here + 200ms backoff from cache TTL).

https://github.com/informalsystems/ibc-rs/blob/e3d0b108831629e5f7fb541c06396e345af0228c/relayer/src/cache.rs#L20

Reply:
Makes sense, let's leave it as-is then!